backend/routes/Terraform.ts

```diff
@@ -289,10 +276,13 @@ router.post('/resources', async (req: ResourcesRequest, res) => {
 
 async function getTerraformFileBufferListFromZip(zipBuffer: Buffer, resourceLocation: string) {
     const resourceLocationTrimmed = trimStart(resourceLocation, '/');
-    const acceptableFilePaths = getTerraformFilePaths(resourceLocationTrimmed);
 
     const files = await decompress(zipBuffer);
-    return files.filter(file => acceptableFilePaths.includes(`${file.path}`)).map(file => file?.data);
+    return files
+        .filter(file => {
+            return file.type === 'file' && isTerraformFilePath(file.path, resourceLocationTrimmed);
+        })
```

Comment:
I think you can safely shorten it to:
```suggestion
        .filter(file => file.type === 'file' && isTerraformFilePath(file.path, resourceLocationTrimmed))
```